content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -168,35 +212,260 @@ kube-apiserver, but this is not a standard.
 
-## Normal user
+
+## Approval or rejection  {#approval-rejection}
+
+Before a [signer](#signers) issues a certificate based on a CertificateSigningRequest,
+the signer typically checks that the issuance for that CSR has been _approved_.
+
+### Control plane automated approval {#approval-rejection-control-plane}
+
+The kube-controller-manager ships with a built-in approver for certificates with
+a signerName of `kubernetes.io/kube-apiserver-client-kubelet` that delegates various
+permissions on CSRs for node credentials to authorization.
+The kube-controller-manager POSTs SubjectAccessReview resources to the API server
+in order to check authorization for certificate approval.
+
+### Approval or rejection using `kubectl` {#approval-rejection-kubectl}
+
+A Kubernetes administrator (with appropriate permissions) can manually approve
+(or deny) CertificateSigningRequests by using the `kubectl certificate
+approve` and `kubectl certificate deny` commands.
+
+To approve a CSR with kubectl:
+
+```shell
+kubectl certificate approve <certificate-signing-request-name>
+```
+
+Likewise, to deny a CSR:
+
+```shell
+kubectl certificate deny <certificate-signing-request-name>
+```
+
+### Approval or rejection using the Kubernetes API {#approval-rejection-api-client}
+
+Users of the REST API can approve CSRs by submitting an UPDATE request to the `approval`
+subresource of the CSR to be approved. For example, you could write an
+{{< glossary_tooltip term_id="operator-pattern" text="operator" >}} that watches for a particular
+kind of CSR and then sends an UPDATE to approve them.
+
+When you make an approval or rejection request, set either the `Approved` or `Denied`
+status condition based on the state you determine:
+
+For `Approved` CSRs:
+
+```yaml
+apiVersion: certificates.k8s.io/v1
+kind: CertificateSigningRequest
+...
+status:
+  conditions:
+  - lastUpdateTime: "2020-02-08T11:37:35Z"
+    lastTransitionTime: "2020-02-08T11:37:35Z"
+    message: Approved by my custom approver controller
+    reason: ApprovedByMyPolicy # You can set this to any string
+    type: Approved
+```
+
+For `Denied` CSRs:
+
+```yaml
+apiVersion: certificates.k8s.io/v1
+kind: CertificateSigningRequest
+...
+status:
+  conditions:
+  - lastUpdateTime: "2020-02-08T11:37:35Z"
+    lastTransitionTime: "2020-02-08T11:37:35Z"
+    message: Denied by my custom approver controller
+    reason: DeniedByMyPolicy # You can set this to any string
+    type: Denied
+```
+
+It's usual to set `status.conditions.reason` to a machine-friendly reason
+code using TitleCase; this is a convention but you can set it to anything
+you like. If you want to add a note for human consumption, use the
+`status.conditions.message` field.
+
+
+## Cluster trust bundles {#cluster-trust-bundles}
+
+{{< feature-state for_k8s_version="v1.27" state="alpha" >}}
+
+{{< note >}}
+In Kubernetes {{< skew currentVersion >}}, you must enable the `ClusterTrustBundles`
```

Comment:
In the Netlify preview, `{{< skew currentVersion >}}`, renders as v1.26 but it'll look right once merged into _dev-1.27_. You can test this by simulating that merge locally and previewing the result.